docs/usage.md

```diff
@@ -16,7 +16,7 @@ simple as executing `watchmaker`. Below is the output of `watchmaker --help`,
 showing the CLI options.
 
 ```shell
-# watchmaker --help
+$ watchmaker --help
```

Comment:
Rationale for changing this? watchmaker needs to execute as root or it won't have perms to do much of anything...

Reply:
The shell code block greys it out as a comment. But, now it's confusing for a whole new reason. Maybe we just get rid of both?

Reply:
I'd say if changing to `$` then add `sudo` also. Otherwise, this change and the one below it are not really related to the sphinx warnings so just remove them from this pr. Can readdress later if we think of a better way.